### PR TITLE
Add Processor Support

### PR DESCRIPTION
**Pull request by Linu-Elias**
## Proposed commit message
Adding Processor Support for the following Packages

- ActiveMQ
- Apache
- Apache Spark
- AWSFargate
- Azure Application Insights
- Azure Billing
- Azure Functions
- Azure OpenAI
- Cassandra
- CockroachDB

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Relates #10393

### CODE REVIEW

File: packages/activemq/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.3.1
+  changes:
+    - description: Add processor support for ActiveMQ broker, queue and topic.
+      type: bugfix
```
Comment (harnish-elastic) [inline packages/activemq/changelog.yml:5]: This should be an enhancement and not a bugfix. As well the minor version should be bump up. That means from `1.3.0` to `1.4.0`.

**Review by harnish-elastic — CHANGES_REQUESTED** [review]

File: packages/apache_spark/data_stream/application/agent/stream/stream.yml.hbs [hunk 2]
```diff
@@ -19,3 +19,8 @@ jmx.mappings:
     attributes:
     - attr: Value
       field: application.status
+
+{{#if processors}}
```
Comment (harnish-elastic) [inline packages/apache_spark/data_stream/application/agent/stream/stream.yml.hbs:23]: Can we move this block before `jmx.mappings:` for better readability? If yes please do for all the packages those containing `jmx.mappings:`

**Review by harnish-elastic — CHANGES_REQUESTED** [review]

File: packages/activemq/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 1.3.1
+  changes:
+    - description: Add processor support for ActiveMQ broker, queue and topic.
```
Comment (harnish-elastic) [inline packages/activemq/changelog.yml:4]: Do we really need package names in the changelog.yml? Can we update the description as per below for all the packages?

`Add processor support for broker, queue and topic data streams.`

File: packages/activemq/changelog.yml (hunk 1 above)
Comment (Linu-Elias) [reply packages/activemq/changelog.yml:5]: we have an issue raised for this - #10393

Comment (harnish-elastic) [reply packages/activemq/changelog.yml:5]: Yes I have checked that. Also please refer the similar PR changelog.yml https://github.com/elastic/integrations/pull/10068/files#diff-00f3cf7e7bdc77301b85e76a90835694b3cf5490056f5ec891b4086a1376ad92R2-R6

File: packages/activemq/changelog.yml (hunk 3 above)
Comment (Linu-Elias) [reply packages/activemq/changelog.yml]: Done

File: packages/apache_spark/data_stream/application/agent/stream/stream.yml.hbs (hunk 2 above)
Comment (Linu-Elias) [reply packages/apache_spark/data_stream/application/agent/stream/stream.yml.hbs]: Done

File: packages/activemq/changelog.yml (hunk 1 above)
Comment (Linu-Elias) [reply packages/activemq/changelog.yml]: Done

**Review by harnish-elastic — APPROVED** [review]
LGTM! 🚀

**Comment by Linu-Elias** [comment]
@elastic/obs-infraobs-integrations hey, could someone take a look at this PR please?

**Review by devamanv — APPROVED** [review]

**Pull request merged**